### PR TITLE
Add 2.5.6

### PR DESCRIPTION
**Pull request by diavrank**
This version includes the binaries for fibers 5.0.1 which fix the problem we had in **2.5.1** for this docker image. So, this PR also includes **Mac M1 Support**. 

Let's try the CI tests with this new version.

More info [here](https://github.com/meteor/meteor/issues/11791#issuecomment-1021242192).

### CODE REVIEW

**Comment by diavrank** [comment]
It seems to be there was a failure with 1.10.2 :( .
https://github.com/disney/meteor-base/runs/4939024347?check_suite_focus=true

Exclude versions 1.X, would it solve it? Maybe it is an error related to this [issue](https://github.com/meteor/meteor/pull/11832).

**Comment by GeoffreyBooth** [comment]
CI stops as soon as any of the concurrent jobs fail. It just happened to fail on 1.10.2 first for you; it failed on 1.9 for me.

It seems to be some issue related to headless Chrome. Maybe the newest version of headless Chrome or puppeteer has a breaking change, or doesn’t work in whatever version of Node the test environment is using?

File: .github/workflows/continuous-integration-workflow.yml [hunk 1]
```diff
@@ -37,6 +37,7 @@ jobs:
           - '2.3.6'
           - '2.4'
           - '2.5'
+          - '2.5.6'
```
Comment (GeoffreyBooth) [inline .github/workflows/continuous-integration-workflow.yml:40]: Please add the skipped versions as comments (see 1.10 above)

File: test.sh [hunk 2]
```diff
@@ -87,9 +87,13 @@ for version in "${meteor_versions[@]}"; do
 	elif [[ "${version}" == 2.3.6 ]] || [[ "${version}" == 2.4 ]]; then
 		node_version='14.17.6'
 
-	# Versions >= 2.5 need Node 14.18.1
-	else
+	# Version 2.5 needs Node 14.18.1
+	elif [[ "${version}" == 2.5 ]]; then
 		node_version='14.18.1'
+
```
Comment (GeoffreyBooth) [inline test.sh:93]: Please add a comment noting which versions are skipped and therefore unsupported.

File: versions.sh [hunk 3]
```diff
@@ -26,5 +26,6 @@ meteor_versions=( \
 	'2.3.5' \
 	'2.3.6' \
 	'2.4' \
-	'2.5'
+	'2.5' \
+	'2.5.6'
```
Comment (GeoffreyBooth) [inline versions.sh:30]: Please comments for the skipped versions (see 1.10 above).

File: .github/workflows/continuous-integration-workflow.yml (hunk 1 above)
Comment (diavrank) [reply .github/workflows/continuous-integration-workflow.yml:45]: Sure. https://github.com/disney/meteor-base/pull/100/commits/8c0306f2ed5152a448609e854ee5482481b2bd88

File: test.sh (hunk 2 above)
Comment (diavrank) [reply test.sh:93]: Thanks! https://github.com/disney/meteor-base/pull/100/commits/8c0306f2ed5152a448609e854ee5482481b2bd88

File: versions.sh (hunk 3 above)
Comment (diavrank) [reply versions.sh:35]: Done. https://github.com/disney/meteor-base/pull/100/commits/8c0306f2ed5152a448609e854ee5482481b2bd88

**Comment by diavrank** [comment]
I'm going to update my branch with main branch to have Puppeter updated.

**Comment by GeoffreyBooth** [comment]
> I’m going to update my branch with main branch to have Puppeteer updated.

Looks like that did the trick! Thanks.

**Pull request merged**